README.md

```diff
@@ -74,6 +76,15 @@ vars:
 ### (Optional) Step 4: Additional configurations
 <details open><summary>Expand/Collapse details</summary>
 
+#### Enabling models that are disabled by default
+For certain models, we have disabled them by default because of a smaller percentages of accounts syncing the underlying tables. To enable them, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+vars:
+    snapchat__using_campaign_country_report: true # false by default. Requires the campaign_geo_country_daily_report
+    snapchat__using_campaign_region_report: true # false by default. Requires the campaign_geo_region_daily_report
```

Comment:
Same request to use the `snapchat_ads__` prefix instead of `snapchat__`

Reply:
updated